blankwindow.py

```diff
@@ -13,7 +13,7 @@
 bg_surf=pygame.image.load('./graphics/background.png').convert()
 
 #importing text
-font=pygame.font.Font('./graphics/subatomic.ttf', 50)
+font=pygame.font.Font('../graphics/subatomic.ttf', 50)
```

Comment:
The file path seems to be incorrect. As classes and graphics are in one directory, we need not add .. two dots before finding the graphics folder. Please revise all of the paths for targets,bullets,background,and font.

Reply:
Adding . one dot is giving me an error. I am adding two dots here as we are going one folder up. Classes and graphics are both in the same directory. But I add ..dots to go one folder up into the graphics folder from the classes folder